cephfs/file_ops.go

```diff
@@ -60,3 +60,35 @@ func (mount *MountInfo) Futime(fd int, times *Utime) error {
 	ret := C.ceph_futime(mount.mount, cFd, uTimeBuf)
 	return getError(ret)
 }
+
+// Timeval struct is the go equivalent of C.struct_timeval type
+type Timeval struct {
+	// Sec represents seconds
+	Sec int64
+	// USec represents microseconds
+	USec int64
+}
+
+// Futimes changes file/directory last access and modification times, here times param
+// is an array of Timeval struct type having length 2.
+//
+// Implements:
+//
+//	int ceph_futimes(struct ceph_mount_info *cmount, int fd, struct timeval times[2]);
+func (mount *MountInfo) Futimes(fd int, times [2]Timeval) error {
```

Comment:
Similar comment to my feedback on #874 - since other APIs use a slice instead of a 2-element array I think we should stick to that consistently.